tests/validation/tests/v3_api/test_cluster_templates.py

```diff
@@ -361,6 +361,159 @@ def test_cluster_template_add_readonly_member():
     cluster_cleanup(standard_newuser_client, cluster)
 
 
+def test_cluster_template_export():
+
+    # Create a DO cluster using rke config. Save a rketemplate from this
+    # cluster (with template name and revision V1).
+    # Create another cluster using the cluster template revision V1
+
+    k8sversionlist = get_k8s_versionlist()
+    standard_user_client = \
+        get_client_for_token(user_token["user_standard"]["token"])
```

Comment:
Rename `user_standard` to reflect the fact that this is a user with create_rke_template rolebinding.
Rename `newuser_standard` to  reflect the fact that this is a standard user

Reply:
Renaming done